lib/v2/lajornada/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/:date/:category?', require('./index'));
```

Comment:
```suggestion
    router.get('/:date?/:category?', require('./index'));
```
Look like both `date` and `category` are optional parameters